tests/tests/optionsTests.robot

```diff
@@ -124,4 +125,63 @@ Verify Output log to a file option
+    List Selection Should Be    collisionResolutionOptions    rename
+    Click Load Button
+    Click Extract And Save Button    ${extraction_timeout}
+    Wait Until Created    ${downloaded_file_path}
+    Validate and Unzip Test File    ${downloaded_file_path}
+    ${ListFiles}  List Files In Directory   ${DOWNLOAD_PATH}${extraction_filter}[archive_name]/app
+    Should Be Equal As Strings     ${ListFiles}    ['MyProg.chm', 'MyProg.exe', 'Readme.txt']
+    Remove File    ${downloaded_file_path}
+    Remove Directory    ${DOWNLOAD_PATH}${extraction_filter}[archive_name]    True
+    Select From List By Index    extractionLanguageFilterOptions    1
+    List Selection Should Be    extractionLanguageFilterOptions    all
+    Select From List By Index    collisionResolutionOptions    1
+    List Selection Should Be    collisionResolutionOptions    rename
+    Click Load Button
+    Click Extract And Save Button    ${extraction_timeout}
+    Wait Until Created    ${downloaded_file_path}
+    Validate and Unzip Test File    ${downloaded_file_path}
+    ${ListFiles}  List Files In Directory   ${DOWNLOAD_PATH}${extraction_filter}[archive_name]/app
+    Should Be Equal As Strings     ${ListFiles}    ['MyProg.chm', 'MyProg.exe', 'Readme.txt', 'Readme.txt@de', 'Readme.txt@nl']
+    Remove File    ${downloaded_file_path}
+    Remove Directory    ${DOWNLOAD_PATH}${extraction_filter}[archive_name]    True
+    Select From List By Index    extractionLanguageFilterOptions    2
+    List Selection Should Be    extractionLanguageFilterOptions    lang
+    Select From List By Index    collisionResolutionOptions    1
+    List Selection Should Be    collisionResolutionOptions    rename
+    Click Load Button
+    Click Extract And Save Button    ${extraction_timeout}
+    Wait Until Created    ${downloaded_file_path}
+    Validate and Unzip Test File    ${downloaded_file_path}
+    ${ListFiles}  List Files In Directory   ${DOWNLOAD_PATH}${extraction_filter}[archive_name]/app
+    Should Be Equal As Strings     ${ListFiles}    ['MyProg.chm', 'Readme.txt']
+    Remove File    ${downloaded_file_path}
+    Remove Directory    ${DOWNLOAD_PATH}${extraction_filter}[archive_name]    True
+    Select From List By Index    extractionLanguageFilterOptions    3
+    List Selection Should Be    extractionLanguageFilterOptions    lang-agn
+    Select From List By Index    collisionResolutionOptions    1
+    List Selection Should Be    collisionResolutionOptions    rename
+    Click Load Button
+    Click Extract And Save Button    ${extraction_timeout}
+    Wait Until Created    ${downloaded_file_path}
+    Validate and Unzip Test File    ${downloaded_file_path}
+    ${ListFiles}  List Files In Directory   ${DOWNLOAD_PATH}${extraction_filter}[archive_name]/app
+    Should Be Equal As Strings     ${ListFiles}    ['MyProg.exe']
+    Remove File    ${downloaded_file_path}
+    Remove Directory    ${DOWNLOAD_PATH}${extraction_filter}[archive_name]    True
```

Comment:
General comment: I think it would be better to split this test to a few smaller test cases - it will be easier to debug and maintenance. We can consult this idea with @krzysztof-rodzaj-mobica. Each value for filter option can be tested by separate scenario.

In case this test is left as it is:
    Select From List By Index    collisionResolutionOptions    1
    List Selection Should Be    collisionResolutionOptions    rename

can be use only once (at the beginning) since there is no page refresh, this setup won't change during test.

In case we decide to split this test, it would be good to set different language then English in some cases.

Reply:
Test was split to a few smaller scenarios.